pkg/providers/tinkerbell/tinkerbell.go

```diff
@@ -16,6 +16,13 @@ import (
 	"github.com/aws/eks-anywhere/pkg/providers"
 	"github.com/aws/eks-anywhere/pkg/templater"
 	"github.com/aws/eks-anywhere/pkg/types"
+	releasev1alpha1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+const (
+	tinkerbellCertURLKey  = "TINKERBELL_CERT_URL"
+	tinkerbellGRPCAuthKey = "TINKERBELL_GRPC_AUTHORITY"
+	tinkerbellIPKey       = "TINKERBELL_IP"
```

Comment:
Where is the PBnJ GRPC info?

Reply:
Would it be okay if I add the flag for PBnJ GRPC while adding changes of generate cluster config file for Tinkerbell? I did not add that flag in this PR because we don't need that flag for clusterctl init infrastructure command.